<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Ciudades</class>
 <widget class="QWidget" name="Ciudades">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>722</width>
    <height>579</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="etiquetaPortugal">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/portugal90.gif) 0 0 0 0 stretch stretch;
border-bottom-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="etiquetaEspana">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/espana.png) 0 0 0 0 stretch stretch;
border-bottom-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="etiquetaFrancia">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/france68.gif) 0 0 0 0 stretch stretch;
border-bottom-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="etiquetaSuiza">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/suiza.png) 0 0 0 0 stretch stretch;
border-bottom-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="etiquetaGranBretana">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/granBretana.gif) 0 0 0 0 stretch stretch;
border-bottom-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="etiquetaHolanda">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/holanda.png) 0 0 0 0 stretch stretch;
border-bottom-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="etiquetaTurquia">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/turquie39.gif) 0 0 0 0 stretch stretch;
border-bottom-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="etiquetaBelgica">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/belgica.png) 0 0 0 0 stretch stretch;
border-bottom-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="etiquetaItalia">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/italia.png) 0 0 0 0 stretch stretch;
border-bottom-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="etiquetaGrecia">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/grecia.png) 0 0 0 0 stretch stretch;
border-bottom-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="etiquetaRepublicaCheca">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/tchequie72.gif) 0 0 0 0 stretch stretch;
border-bottom-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="etiquetaPolonia">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pologne20.gif) 0 0 0 0 stretch stretch;
border-bottom-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="etiquetaRumania">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/roumanie23.gif) 0 0 0 0 stretch stretch;
border-bottom-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="etiquetaUcrania">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/ukraine69.gif) 0 0 0 0 stretch stretch;
border-bottom-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="etiquetaBulgaria">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/bulgaria.png) 0 0 0 0 stretch stretch;
border-bottom-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="etiquetaFinlandia">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/finlande19.gif) 0 0 0 0 stretch stretch;
border-bottom-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="etiquetaSuecia">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/suecia.png) 0 0 0 0 stretch stretch;
border-bottom-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="etiquetaAlemania">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>641</width>
     <height>541</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/alemania.png) 0 0 0 0 stretch stretch;
border-bottom-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(0, 0, 0, 255), stop:1 rgba(255, 255, 255, 255));</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>etiquetaPortugal</zorder>
  <zorder>etiquetaSuiza</zorder>
  <zorder>etiquetaGranBretana</zorder>
  <zorder>etiquetaHolanda</zorder>
  <zorder>etiquetaTurquia</zorder>
  <zorder>etiquetaBelgica</zorder>
  <zorder>etiquetaFrancia</zorder>
  <zorder>etiquetaItalia</zorder>
  <zorder>etiquetaGrecia</zorder>
  <zorder>etiquetaRepublicaCheca</zorder>
  <zorder>etiquetaPolonia</zorder>
  <zorder>etiquetaRumania</zorder>
  <zorder>etiquetaUcrania</zorder>
  <zorder>etiquetaFinlandia</zorder>
  <zorder>etiquetaSuecia</zorder>
  <zorder>etiquetaAlemania</zorder>
  <zorder>etiquetaEspana</zorder>
  <zorder>etiquetaBulgaria</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
